.github/workflows/main.yml

```diff
@@ -85,3 +85,28 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+
+  trigger_endless-key-app:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/') || startsWith(inputs.tagname, 'v')
+    needs: [build_apps-bundle, build_kolibri-explore-plugin_whl]
+
+    steps:
+      - name: Get tag name as the release version when at a tag
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          echo "VERSION=${{ github.ref_name }}" >> $GITHUB_ENV
+
+      - name: Get tag name as the release version from input
+        if: ${{ !startsWith(github.ref, 'refs/tags/') && startsWith(inputs.tagname, 'v') }
+        run: |
+          echo "VERSION=${{ inputs.tagname }}" >> $GITHUB_ENV
+
+      - name: Trigger building endless-key-app
+        run: |
+          curl -H "Accept: application/vnd.github.everest-preview+json" \
+               -H "Authorization: token ${{ secrets.BUILD_ENDLESSKEYAPP_TOKEN }}" \
+               --request POST \
+               --data '{"event_type": "kolibri-explore-plugin", "client_payload": { "VERSION": "$VERSION"}}' \
+               https://api.github.com/repos/endlessm/endless-key-app/dispatches
```

Comment:
I was looking around for how to do this and found [this comment](https://github.com/orgs/community/discussions/26323#discussioncomment-3251451) suggesting to use the [github-script action](https://github.com/actions/github-script). That seems a little nicer since it knows how to construct GitHub API requests with proper authentication. Then you'd use the [`rest.repos.createDispatchEvent`](https://octokit.github.io/rest.js/v19#repos-create-dispatch-event) API.

Reply:
That looks nice with the [token](https://github.com/actions/github-script#using-a-separate-github-token)!

Reply:
Updated by the new commit.